data/kesans.mdx

```diff
@@ -0,0 +1,12 @@
+---
+name: Kesans 
+description:We are a small company that engages in solving problems of commons that software can address. Currently, we are working on developing Kanakbooks, a cloud-based accounting/billing/e-commerce software that is supposed to be affordable for everyone. 
```

Comment:
Sorry for being late @vinodgm, was going through some health issues lately. Anyway there's a build error caused since there is no space b/w `description:` and the content It should be like this

`description: We are a small company that engages in solving problems of commons that software can address. `

Reply:
Thanks for the reply. My bad... Added that whitespace. But still seems to have some issue with build. Not sure what it is..

Reply:
There's a small build error which I think is not because of this. Will check it out ASAP. Thank for contributing